conf/application.conf

```diff
@@ -104,6 +104,11 @@ microservice {
         port = 9583
       }
 
+      pdf_generator_service {
```

Comment:
have you added this dependency to our `service-manager-config` definitions?

Reply:
No.  I wanted to see if it would run "out of the box" with no config changes.   I had an experiment with sm config but there appears to be no way to change config for a service.

Reply:
PR created and merged